mindmeld/core.py

```diff
@@ -1059,6 +1059,21 @@ def to_dict(self):
         return base
 
 
+class Registry:
+    """A registration class to map object names to corresponding objects."""
+    def __init__(self):
+        self.func_registry = {}
+
+    def add_to_func_registry(self, func_name, func):
+        """Populates the function registry map.
+
+        Args:
+            func_name (str): Name to be used as key for the function.
+            func (func): Callable function.
+        """
+        self.func_registry[func_name] = func
```

Comment:
Its probably a good idea to implement a getter here instead of exposing the underlying array DS to the client functions